### ISSUE TITLE
clip with HDR

### ISSUE THREAD

**Issue by RBR8man** [Ajatt-Tools/videoclip#26]
The tool only works with SDR video
how would I have it clip an HDR one?

**Comment by Cal-Nine** [comment]
Even if it always outputs in SDR, it would be good to have normal looking mapped colors, with a selection of tonemapping alrorithms: https://ayosec.github.io/ffmpeg-filters-docs/7.1/Filters/Video/tonemap.html. Reinhard would be a good default.

**Comment by AnimeAjay** [comment]
Yeah, I would very much like for it to tonemap. Even if it's just a toggle we can set ourselves rather than automatic detection, it'd be great to include.

**Comment by tatsumoto-ren** [comment]
Please create a pull request once you finish!

**Comment by AnimeAjay** [comment]
This was a request, not a proclamation of any ability to add it personally, but I think you knew that.

**Comment by tatsumoto-ren** [comment]
I hope someone volunteers to add it.
